dango/types/src/dex/events.rs

```diff
@@ -1,4 +1,5 @@
 use {
+    super::PairId,
     crate::dex::{Direction, OrderId},
```

Comment:
I don't like having `super` and `crate` in the same `use` statement

```suggestion
    crate::dex::{Direction, OrderId, PairId},
```